test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -435,11 +435,12 @@ def setUpClass(cls):
         if cls.use_cur_setup:
             _, path = tempfile.mkstemp(prefix="saved_custom_setup",
                                        suffix=".json")
-            cls.saved_cus_setup = cls.server.save_configuration(path, 'w')
-            if cls.saved_cus_setup:
+            ret = cls.server.save_configuration(path, 'w')
+            if ret:
                 cls.saved_file = path
             else:
-                self.logger.error("Failed to save custom setup")
+                cls.logger.error("Failed to save custom setup")
+                sys.exit(1)
```

Comment:
I think you should do self.fail("Failed to save custom setup") instead of doing sys.exit(1), that way pbs_benchpress can exit gracefully. Same comment elsewhere in this file

Reply:
`self.fail` is only available in test case code, not in this class, you can raise an exception to stop execution flow here